packages/@ember/-internals/glimmer/tests/integration/helpers/unbound-test.js

```diff
@@ -512,7 +512,7 @@ moduleFor(
           this.removeObserver('value.lastName', this, this.recompute);
         },
         compute([value]) {
-          if (this.get('value')) {
```

Comment:
TL;DR: They would have triggered the deprecation since we're consuming and then immediately setting the value (`get` consumes). I added an explicit test for this, and since `get` wasn't integral to the test here, figured we could remove it here.